Update InstalledSoftware.csv

mtreanor-r7 opened the pull request:
adding in data transfer categories

mtreanor-r7 committed:
Update InstalledSoftware.csv

adding in data transfer categories

mgreen27 commented on csv/InstalledSoftware.csv:
```diff
@@ -1,4 +1,19 @@
 Category,Regex,Comment
+Data Transfer,rclone,
+Data Transfer,megasync,
+Data Transfer,plink,
```
plink is part of the PuTTY suite so you probably wont see it in installed applications.

mtreanor-r7 replied:
No problem, happy for this to be committed and removed

mgreen27 committed:
Update InstalledSoftware.csv

changed plink to putty. Need to also monitor some of the other entries for effectiveness.
